tools/render-sample-sites.py

```diff
@@ -29,9 +27,8 @@ def worker(theme):
             shutil.rmtree(build_location)
         build_location.mkdir(parents=True)
 
-        # Install required packages
-        packages = sorted({theme.pypi})  # prevents duplication
-        env.install("--pre", f"sphinx=={SPHINX_VERSION}", *packages)
+        env.install("--pre", f"sphinx")  # we test against the latest Sphinx version
+        env.install(theme.pypi)
```

Comment:
Note that this means that if the theme specifies its dependencies like `sphinx<4`, it wouldn't be built against the latest pre-release, but that's fine since the theme has an explicit pin.